web/src/app/workplace/tasklist/tasklist.component.scss

```diff
@@ -1,3 +1,52 @@
 .clickable {
     cursor: pointer;
 }
+
+.task-list-full-height{
```

Comment:
We are using .workbasket-list-full-height, task-list-full-height and workbasket-list-full-height with the same rule in three different parts of the code, maybe we can extract this into master-and-detail.component, probably should extract all common  sass into parent sass